front/src/modules/ui/navigation/mobile-navbar/MobileNavBar.tsx

```diff
@@ -0,0 +1,34 @@
+import { useLocation } from 'react-router-dom';
+
+import { Favorites } from '@/favorites/components/Favorites';
+import { SettingsNavbar } from '@/settings/components/SettingsNavbar';
+import { IconBuildingSkyscraper } from '@/ui/display/icon/index';
+import { useIsSubMenuNavbarDisplayed } from '@/ui/layout/hooks/useIsSubMenuNavbarDisplayed';
+import MainNavbar from '@/ui/navigation/desktop-navbar/components/MainNavbar';
+import NavItem from '@/ui/navigation/desktop-navbar/components/NavItem';
+import NavTitle from '@/ui/navigation/desktop-navbar/components/NavTitle';
+
+export const MobileNavBar = () => {
+  const currentPath = useLocation().pathname;
+
+  const isInSubMenu = useIsSubMenuNavbarDisplayed();
+
+  return (
+    <>
+      {!isInSubMenu ? (
+        <MainNavbar>
+          <Favorites />
+          <NavTitle label="Workspace" />
+          <NavItem
```

Comment:
Let's share items between navbars (otherwise the day someone adds something to the menu, there's a 50% chance they forget adding it to the mobile version 😁)